### DISCUSSION TITLE
Welcome to Paddle Quantum Discussions!

### DISCUSSION THREAD

**Discussion by Quleaf** [PaddlePaddle/Quantum#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oum-git** [comment]
Hello, I am using paddle quantum on a gpu (paddle paddle gpu). After initializing my circuit using ansatz.circuit, i always obtain a circuit with a unitary-matrix with values equal to zero which is not the case when using a cpu.  Can you help me? thanks.
